projects/storefrontlib/src/cms-components/user/checkout-login/checkout-login.component.ts

```diff
@@ -1,71 +1,48 @@
 import { Component, OnDestroy } from '@angular/core';
-import {
-  AbstractControl,
-  FormBuilder,
-  FormGroup,
-  Validators,
-} from '@angular/forms';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { ActiveCartService, AuthRedirectService } from '@spartacus/core';
 import { Subscription } from 'rxjs';
-import { FormUtils } from '../../../shared/utils/forms/form-utils';
 import { CustomFormValidators } from '../../../shared/utils/validators/custom-form-validators';
 
 @Component({
   selector: 'cx-checkout-login',
   templateUrl: './checkout-login.component.html',
 })
 export class CheckoutLoginComponent implements OnDestroy {
```

Comment:
Breaking changes info needed:
- removed isNotValid and isEmailConfirmInvalid

Reply:
Done